### PR TITLE
style: US-3 Update the bg view on calculator and math op list

### PR DESCRIPTION
**Pull request by JessCasasDev**

### CODE REVIEW

File: src/environments/environment.ts [hunk 1]
```diff
@@ -14,3 +14,4 @@ export const environment = {
  * on performance if an error is thrown.
  */
 // import 'zone.js/dist/zone-error';  // Included with Angular CLI.
+export const BASE_URL = 'http://localhost:3000';
```
Comment (JuanSeBestia) [inline src/environments/environment.ts:17]: we can use enviroment.prod to get BASE_URL of prod?

**Comment by JuanSeBestia** [comment]
What do you think if we include a commit in semantic-release [https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines)

File: src/environments/environment.ts (hunk 1 above)
Comment (JessCasasDev) [reply src/environments/environment.ts:17]: *Can we use...

But there isn't a BASE_URLon prod yet...

**Comment by JessCasasDev** [comment]
@JuanSeBestia I've updated the commit message, is that right?

File: src/environments/environment.ts (hunk 1 above)
Comment (JuanSeBestia) [reply src/environments/environment.ts:17]: You are rigth :(

**Comment by JuanSeBestia** [comment]
> @JuanSeBestia I've updated the commit message, is that right?

Yes and not, the title of PR now are OK, but you would include a commit with a similar syntax.

Right now your commit say "improve view", the correct commit is "styles(US-3): Improve view on calculator and math op"

You can add a empty commit with comand
git commit --allow-empty -m "styles(US-3): Improve view on calculator and math op"

Or renaming de commit

**Comment by JessCasasDev** [comment]
🤦‍♀🤦‍♀🤦‍♀🤦‍♀🤦‍♀🤦‍♀ I didn't notice I only rename the PR, sorry

**Pull request merged**